index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="container">
+        <a class="logo">My Logo</a>
+
+        <nav class="menu-icon">
+            <img src="src/Icon-Menu.png" alt="menu"/>
+        </nav>
+    </header>
+    <section class="headline">
+
+        <h2 class="title">I’m Mariane Glad to see you!
+            </h2>
```

Comment:
- [ ] Please I would suggest you always use `h1` tag for your main website title, as for this, you can change this h2 to an h1 instead